examples/aws/firewall/firewallConfig.go

```diff
@@ -1,4 +1,4 @@
-package main
+package firewall
```

Comment:
Go programs require a main package. This change breaks the script

Reply:
I think in general we shouldn't propose changes in source code unless we're sure/executed in local and see it's working as expected

Reply:
that was my IDE's brilliant suggestion :(

Reply:
reverted and tested script locally

Reply:
@boranx I agree with that. Also that every piece of code should have a test in the pipeline pre-merge to make sure nothing like that skips our attention.